bitnami/cassandra/README.md

```diff
@@ -142,19 +142,27 @@ The following table lists the configurable parameters of the Cassandra chart and
+| `service.nodePorts.metrics`            | Kubernetes Metrics node port                                                                                            | `""`                                                         |
+| `service.loadBalancerIP`               | LoadBalancerIP if service type is `LoadBalancer`                                                                        | `nil`                                                        |
+| `service.annotations`                  | Annotations for the service                                                                                             | {}                                                           |
+| `networkPolicy.enabled`                | Enable NetworkPolicy                                                                                                    | `false`                                                      |
+| `networkPolicy.allowExternal`          | Don't require client label for connections                                                                              | `true`                                                       |
+| `hostNetwork`                          | Use Host-Network for the PODs (if true, also dnsPolicy: ClusterFirstWithHostNet is set)                                 | `false`                                                      |
+| `containerPorts.cassandra.intra`       | Intra Port on the Host and Container                                                                                    | `7000`                                                       |
+| `containerPorts.cassandra.tls`         | TLS Port on the Host and Container                                                                                      | `7001`                                                       |
+| `containerPorts.cassandra.jmx`         | JMX Port on the Host and Container                                                                                      | `7199`                                                       |
+| `containerPorts.cassandra.cql`         | CQL Port on the Host and Container                                                                                      | `9042`                                                       |
+| `containerPorts.cassandra.thrift`      | Thrift Port on the Host and Container                                                                                   | `9160`                                                       |
+| `containerPorts.metrics.http`          | HTTP Port on the Host and Container                                                                                     | `8080`                                                       |
+| `containerPorts.metrics.jmx`           | JMX Port on the Host and Container                                                                                      | `5555`                                                       |
```

Comment:
I'd move all these `containerPorts` parameters under the "Statefulset parameters" section.

Reply:
Great, many thanks for your support.